### DISCUSSION TITLE
Endpoints of API

### DISCUSSION THREAD

**Discussion by apassan23** [joshidipesh12/rss-swift#2] [Q&A] [answered]
What should be endpoints of the API?

**Comment by joshidipesh12** [comment, marked as answer]
The endpoints can be
- feedsByRss (already done)
> takes rss link as query and returns the parsed list of articles 
- topArticles (or trending)
> this can randomly parse 5-10 rss links from the list and return list ( for landing page too )

Keep appending ideas 💡
